suricata.yaml.in

```diff
@@ -1017,8 +1018,6 @@ app-layer:
         content-inspect-window: 4096
     imap:
       enabled: detection-only
-    pop3:
```

Comment:
there should be a pop3 section here

Reply:
What all should be here? Are some of these fields generated automatically?

Reply:
minimum is something like
```
    pop3:
      enabled: yes
```
Then depending on features supported there can be `stream-depth` override, `max-tx` setting, etc.